api/src/restful.js

```diff
@@ -3,24 +3,31 @@ import bodyParser from 'body-parser';
 
 import * as reports from './reports';
 import * as updates from './updates';
+import * as log from './log';
 
 export default function createRouter(db) {
   const router = express.Router();
   const reportsColl = db.collection('reports');
   const updatesColl = db.collection('updates');
+  const logColl = db.collection('log');
 
   router.use(cors);
   router.use(bodyParser.json());
 
   router.route('/reports').get(reports.get(reportsColl));
   router.route('/reports/current').get(reports.current(reportsColl));
   router.route('/reports').post(reports.create(reportsColl));
+  router.route('/reports/remove/:id').post(reports.remove(reportsColl));
 
   router.route('/updates').get(updates.get(updatesColl));
   router.route('/updates').post(updates.create(updatesColl));
+  router.route('/updates/remove/:id').post(updates.remove(updatesColl));
   router.route('/updates/:id').post(updates.update(updatesColl));
   router.route('/resolve').post(updates.resolve(updatesColl));
 
+  router.route('/log').get(log.get(logColl));
+  router.route('/log').post(log.insert(logColl));
```

Comment:
No, I think #10 is enabled by aggregating the `updates` collection.  I have a WIP.

I'd like to avoid increasing the complexity of the code for as long as possible.  Also, let's try to avoid over-engineering the code by designing for future requirements.  If an in-memory array serves the current purpose well, I say let's go with it, please.  It's simple and synchronous.